run-tests.pl

```diff
@@ -34,17 +35,22 @@
       : undef;
 });
 
+my $plugin_dir = $ENV{"SYTEST_PLUGINS"} || "plugins"; # Read plugin dir from env var SYTEST_PLUGINS or fallback to plugins
```

Comment:
please could you document this environment variable in https://github.com/matrix-org/sytest/#running ?